### PR TITLE
Add content - energy infrastructure framing questions

### PR DESCRIPTION
**Pull request by nickumia-reisys**
Related to
- https://github.com/GSA/data.gov/issues/3891

Replaces
- https://github.com/GSA/datagov-website/blob/main/www.data.gov/energy-infrastruture-framing-questions.md

### CODE REVIEW

**Comment by nickumia-reisys** [comment]
Don't merge this until we figure out if the `back to climate homepage` link is useful.

**Comment by nickumia-reisys** [comment]
@Jin-Sun-tts @btylerburton This is ready to go 🪄

**Review by Jin-Sun-tts — APPROVED** [review]

**Pull request merged**
